docs-internal/api/nu-designer-openapi.yaml

```diff
@@ -602,6 +602,14 @@ paths:
                 Example1:
                   summary: Comment is required
                   value: Comment is required.
+                Example2:
+                  summary: Scenario validation error
+                  value: 'ValidationErrors(Map(filter  -> List(NodeValidationError(ExpressionParserCompilationError,Failed
```

Comment:
Not sure whether it is connected with your change, but is this the intended value?
It looks neither human nor computer readable.